tests/functional/spec/features/hub_and_spoke/hub_and_spoke.spec.js

```diff
@@ -228,6 +241,96 @@ describe("Feature: Hub and Spoke", () => {
     });
   });
 
+  describe("Given a user opens a schema with hub required sections based on a repeating section", () => {
+    beforeEach("Load survey", async () => {
+      await browser.openQuestionnaire("test_hub_section_required_with_repeat.json");
+    });
+
+    it("When the repeating sections are complete. Then the hub should be displayed", async () => {
+      await $(PrimaryPersonListCollectorPage.yes()).click();
+      await $(PrimaryPersonListCollectorPage.submit()).click();
+      await $(PrimaryPersonListCollectorAddPage.firstName()).setValue("Marcus");
+      await $(PrimaryPersonListCollectorAddPage.lastName()).setValue("Twin");
+      await click(PrimaryPersonListCollectorAddPage.submit());
+      await $(ListCollectorPage.no()).click();
+      await $(ListCollectorPage.submit()).click();
+      await click(ListCollectorSectionSummaryPage.submit());
```

Comment:
After this, it is worth trying to access Hub URL and showing it won't go there.

Reply:
Added 👍